opening_hours/hours.py

```diff
@@ -165,7 +165,7 @@ def get_opening_hours(
         for opening_hours in day_data_in["opening_hours"]:
             day_data_out = {
                 "resource_id": day_data_in["resource"]["id"],
-                "origin_id": day_data_in["resource"]["origins"]["origin_id"],
+                "origin_id": day_data_in["resource"]["origins"][0]["origin_id"],
```

Comment:
Hmm wonder if this will cause trouble if querying both tvp and tprek opening hours at once, but i guess the way it's done currently it shouldn't happen.

Reply:
yes, you cannot use the function to fetch two origins

Reply:
Don't se the use case for it now but surely we could implement such feature enhancement later if its needed.